tests/unit/workers/tasks/test_import_insights_from_image.py

```diff
@@ -0,0 +1,149 @@
+                "model": "nutriscore",
+                "confidence": 0.816114128,
+                "bounding_box": [0.591395915, 0.477426708, 0.688566923, 0.624632716],
+            },
+            value_tag="en:nutriscore-a",
+            value="",
+            automatic_processing="f",
+        ),
+        PredictionFactory(
+            type="label",
+            barcode="3302740030949",
+            data={
+                "model": "nutriscore",
+                "confidence": 0.913557172,
+                "bounding_box": [0.596912444, 0.429353863, 0.70557, 0.592592061],
+            },
+            value_tag="en:nutriscore-c",
+            value="",
+            automatic_processing="t",
+        ),
+        PredictionFactory(
+            type="label",
+            barcode="3302740030949",
+            data={
+                "model": "nutriscore",
+                "confidence": 0.99035,
+                "bounding_box": [0.575399518, 0.472130537, 0.691237211, 0.625134587],
+            },
+            value_tag="en:nutriscore-e",
+            value="",
+            automatic_processing="t",
+        ),
+    ]
+
+    product_insight1 = ProductInsightFactory(
+        barcode="3302740030949",
+        value_tag="en:nutriscore-c",
+        type="label",
+        data={
+            "model": "nutriscore",
+            "confidence": 0.816114128,
+            "bounding_box": [0.591395915, 0.477426708, 0.688566923, 0.624632716],
+        },
+    )
+
+    product_insight2 = ProductInsightFactory(
+        barcode="3302740030941",
+        value_tag="en:nutriscore-a",
+        type="label",
+        data={
+            "model": "nutriscore",
+            "confidence": 0.816114128,
+            "bounding_box": [0.591395915, 0.477426708, 0.688566923, 0.624632716],
+        },
+    )
+
+    product_insight3 = ProductInsightFactory(
+        barcode="3302740030942",
+        value_tag="en:salmons",
+        type="label",
+        data={
+            "model": "neural",
+            "confidence": 0.816114128,
+            "bounding_box": [0.591395915, 0.477426708, 0.688566923, 0.624632716],
+        },
+    )
+
+    product_insight3 = ProductInsightFactory(
+        barcode="3302740030943",
+        value_tag="en:nutriscore-c",
+        type="category",
+        data={
+            "model": "nutriscore",
+            "confidence": 0.816114128,
+            "bounding_box": [0.591395915, 0.477426708, 0.688566923, 0.624632716],
+        },
+    )
+
+    barcode = product_insight1.barcode
+
+    product = {
+        "code": barcode,
+        "product": {
+            "labels": "Sans colorants, Nutriscore, Nutriscore C, Fumé au bois de hêtre",
+            "product_name": "Pommes de terre gratinées aux lardons et champignons",
+        },
+        "status": 1,
+        "status_verbose": "product found",
+    }
+
+    get_product_storeactual_predictions_all = [
+        item.to_dict() for item in predictions_all
+    ]
+
+    product_store_mock = mocker.patch(
```

Comment:
Hey @alexgarel 

I am not sure what to match and predict here.  So I have just initialized.